front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/ChartWidgets/VisitorsTrafficSourcesWidget/Table.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { VisitorsTrafficSourcesResponse } from 'api/graph_data_units/responseTypes/VisitorsTrafficSourcesWidget';
+
+import {
+  Box,
+  Table,
+  Thead,
+  Tr,
+  Th,
+  Tbody,
+  Td,
+  colors,
+  stylingConsts,
+} from '@citizenlab/cl2-component-library';
+import messages from 'components/admin/GraphCards/VisitorsTrafficSourcesCard/messages';
+import { FormattedMessage } from 'utils/cl-intl';
+
+interface Props {
+  tableData: VisitorsTrafficSourcesResponse['data']['attributes']['top_50_referrers'];
+}
+
+const TableView = ({ tableData }: Props) => {
+  return (
+    <Box>
+      <Table
+        border={`1px solid ${colors.grey300}`}
+        borderRadius={stylingConsts.borderRadius}
+        innerBorders={{
+          bodyRows: true,
+        }}
+      >
+        <Thead>
+          <Tr background={colors.grey50}>
+            <Th py="16px">
+              <FormattedMessage {...messages.referrer} />
+            </Th>
+            <Th py="16px">
+              <FormattedMessage {...messages.visits} />
+            </Th>
+            <Th py="16px">
+              <FormattedMessage {...messages.visitors} />
+            </Th>
+          </Tr>
+        </Thead>
+        <Tbody>
+          {tableData.slice(0, 5).map((row, i) => (
```

Comment:
For now, we just show a table view of the top 5 referrers. In another PR it will be possible to open this table full screen and see a 'paginated' version with the top 50.